### ISSUE TITLE
Create alt-text for the pie chart graphs on the investments and systems reports

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/GEAR3#207]
In our call with the Section 508 team, while manually reviewing the site with a screenreader, we found that the charts on the investments and systems reports were not easily readable. To remedy this, it was recommended to add alt-text to the chart to better explain what it is conveying.

**Comment by hatfieldjm4** [comment]
Alt text for the pie charts is still not visible for Dan. We will need to revisit this.

**Comment by kinggsagov** [comment]
<a href="#" id="sysVizText"      
      [ariaLabel]="getAriaLabel(vizData)" 
      style="visibility: hidden"
    ></a>
